testing_app/test/perf_test_e2e.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file duplicate the behavior of `test_driver/perf_test.dart` but uses
```

Comment:
nit: "This file duplicates the behavior of `test_driver/perf_test.dart`, but uses the e2e package to implement a host-independent test."

Reply:
Done.